src/OMSimulatorLib/Connector.cpp

```diff
@@ -155,20 +193,21 @@ oms_status_enu_t oms::Connector::exportToSSD(pugi::xml_node &root) const
   switch (this->type)
   {
   case oms_signal_type_boolean:
-    node.append_attribute("type") = "Boolean";
+    node.append_child(oms::ssp::Version1_0::ssc::boolean_type);
     break;
   case oms_signal_type_enum:
-    node.append_attribute("type") = "Enumeration";
+    node.append_child(oms::ssp::Version1_0::ssc::enumeration_type);
     break;
   case oms_signal_type_integer:
-    node.append_attribute("type") = "Integer";
+    node.append_child(oms::ssp::Version1_0::ssc::integer_type);
     break;
   case oms_signal_type_real:
-    node.append_attribute("type") = "Real";
+    node.append_child(oms::ssp::Version1_0::ssc::real_type);
     break;
   case oms_signal_type_string:
-    node.append_attribute("type") = "String";
+    node.append_child(oms::ssp::Version1_0::ssc::string_type);
     break;
+  //TODO handle Binary Types
```

Comment:
What binary type? Please catch all unhandled cases and print an error.

Reply:
@ added log Error to catch the Binary types exceptions in connectors